pkg/sql/sqlbase/table.go

```diff
@@ -2026,6 +2026,12 @@ type ConstraintDetail struct {
 	Details     string
 	Unvalidated bool
 
+	// This fields just a stubs for deferrable constraints logic.
+	// Looks like AlterTableAddConstraint command (as well as CreateTable) should
+	// store somewhere ValidationBehavior for further use in this structure.
+	IsDeferrable      bool
+	InitiallyDeferred bool
```

Comment:
Likewise, I think it would be better not to have these around if they're not actually used. I think supporting `DEFERRABLE` constraints is a moderately large project.

Reply:
Indeed. There's an RFC coming soon on the topic courtesy of @BramGruneir.

Reply:
Ok, stubs removed.